### ISSUE TITLE
Doesn't work with multiple panes

### ISSUE THREAD

**Issue by Ellpeck** [ludovicchabant/obsidian-remember-file-state#5]
Hi, I love this plugin and how simple it is to set up and use!

I'm having an issue with it, however: If you have two files open next to each other in two separate panes (or even two separate windows with the recent Obsidian changes), switching between them using the mouse will **always** return the cursor to the top of the file, even when default Obsidian behavior would keep it in the same place or move it to the location that you clicked on the pane in. 

Thanks so much again for the plugin, and for looking into this issue. ❤️

**Comment by ludovicchabant** [comment]
Hi there! I've seen this bug once and have had issues reproducing it... can you look inside the `.obsidian/plugins` folder (inside your vault) and see if you maybe have two versions of the plugin? (one called `remember-file-state` which should be deleted, and one called `obsidian-remember-file-state`, which should be kept)

**Comment by Ellpeck** [comment]
No, I only have `obsidian-remember-file-state` in my plugins directory. 

For me, the issue is very easy to reproduce by just opening a file in the center pane, and then selecting a different file and opening it in a new pane. If I then select or click on an area in one pane, and then do the same in the other, switching back to the first pane will always place the cursor at the very top of the file.

**Comment by ludovicchabant** [comment]
Thanks for checking. What version of Obsidian, and of the plugin, do you have?

**Comment by ludovicchabant** [comment]
Ok I think I found the problem -- try version 1.0.6? https://github.com/ludovicchabant/obsidian-remember-file-state/releases/tag/1.0.6

Thanks!

**Comment by Ellpeck** [comment]
That seems to have fixed it! Thanks so much ❤️
